src/main/java/uk/gov/hmcts/ccd/domain/model/definition/SearchInputDefinition.java

```diff
@@ -40,4 +41,16 @@ public void setFields(List<SearchInputField> fields) {
         this.fields = fields;
     }
 
+    /**
+     **/
+    @ApiModelProperty(value = "")
```

Comment:
@ApiModelProperty(value = "")  do we need the value="" ?

Reply:
Purely added for consistency with rest of this class.